packages/nocodb/src/plugins/smtp/SMTP.ts

```diff
@@ -23,13 +24,11 @@ export default class SMTP implements IEmailAdapter {
         typeof this.input?.ignoreTLS === 'boolean'
           ? this.input?.ignoreTLS
           : this.input?.ignoreTLS === 'true',
-      auth: {
-        user: this.input?.username,
-        pass: this.input?.password,
-      },
       tls: {
         rejectUnauthorized: this.input?.rejectUnauthorized,
       },
+      ...(this.input?.user ? { auth: { user: this.input?.user } } : {}),
```

Comment:
is it `user` or `username`  ?  As per plugin meta it's `username`

Reply:
![image](https://github.com/nocodb/nocodb/assets/64391274/8722223d-9d5b-4822-83f3-10c229928bb2)
It's user

Reply:
`this.input?.user` => `this.input?.username`